test/test_activity_update.py

```diff
@@ -27,7 +27,6 @@ def test_user_creation(self):
         user = UserFactory.create()
         update_feed = get_update_feed()
         err_msg = "It should be the same user"
-        assert update_feed[0]['id'] == user.id, err_msg
```

Comment:
Good question. I have preferred to remove it, but yes, dunno what we should do. The main issue is that the ID of a user can be linked to a user (as that ID is stored in task_run and results) and a shortname is not. What do you think?